intro-meraki/meraki-03-location-scanning-python/locationscanningreceiver/Dockerfile

```diff
@@ -6,4 +6,5 @@ RUN pip install -r requirements.txt
 
 EXPOSE 5001
 
-CMD ["python", "locationscanningreceiver.py", "-v", "simulator", "-s", "simulator"]
+CMD ["python", "locationscanningreceiver.py", "-v", "5d6d070c526bc5aad1bcf0228e3b1bc4ef5303d0", "-s", "5d6d070c526bc5aad1bcf0228e3b1bc4ef5303d0"]
```

Comment:
Hi, thanks for this pull request, but in this case we want to leave as-is since those values will change over time. I'll close this PR but please reopen if I've misinterpreted.